eng/pipelines/1es-redirect.yml

```diff
@@ -0,0 +1,75 @@
+resources:
+  repositories:
+    - repository: 1ESPipelineTemplates
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+      ref: refs/tags/azure-sdk-build-tools_20230829.1
+    - repository: azure-sdk-tools
+      type: github
+      name: Azure/azure-sdk-tools
+      endpoint: azure
+      ref: refs/heads/main
+    - repository: azure-sdk-for-net
+      type: github
+      name: Azure/azure-sdk-for-net
+      endpoint: azure
+      ref: main
+
+parameters:
+- name: stages
+  type: stageList
+  default: []
+- name: variables
+  type: object
+  default: {}
+- name: Use1ESOfficial
+  type: boolean
+  default: true
+
+extends:
+  ${{ if and(parameters.Use1ESOfficial, eq(variables['System.TeamProject'], 'internal')) }}:
+    template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  ${{ else }}:
+    template: v1/1ES.Unofficial.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    settings:
+      skipBuildTagsForGitHubPullRequests: true
+    sdl:
+      sourceAnalysisPool:
+        name: azsdk-pool-mms-win-2022-general
+        image: azsdk-pool-mms-win-2022-1espt
+        os: windows
+      sourceRepositoriesToScan:
+        exclude:
+          - repository: azure-sdk-build-tools
+      psscriptanalyzer:
+        compiled: true
+        break: true
+
+  stages: 
+  - ${{ each stage in parameters.stages }}:
+    - ${{ each pair in stage }}:                
+        # Insert all properties other than "variables"
+        ${{ if ne(pair.key, 'variables') }}:
+          ${{ pair.key }}: ${{ pair.value }}
+      variables:
+        # Add image variables
+        LINUXPOOL: azsdk-pool-mms-ubuntu-2004-general
```

Comment:
I think defining common image variables in a dedicated file is clearer and more consistent with our patterns.

Reply:
More consistent, sure.   The goal is to allow for variable splatting with defaults.  This would result in a single import of the template in the repository making the need for it to be a nested template unnecessary.